### PR TITLE
feat(azure-db): use managed-pg helm chart

### PR DESCRIPTION
**Pull request by revolunet**

### CODE REVIEW

**Comment by douglasduteil** [comment]
@revolunet  We should change the would job to use the helm chart no ?

**Comment by revolunet** [comment]
indeed but i dont know

**Comment by douglasduteil** [comment]
> indeed but i dont know

You don't know *how ?

**Comment by douglasduteil** [comment]
@revolunet > https://github.com/SocialGouv/gitlab-ci-yml/pull/115

**Comment by revolunet** [comment]
will test this branch asap

**Comment by revolunet** [comment]
except the quoted DB name, creation/drop works nice !

**Comment by revolunet** [comment]
using it it sample-next-app : the sample-next-app-secrets are not duplicated on each new namespace

**Comment by revolunet** [comment]
i can confirm this works nicely with helm-charts@6.1-alpha

File: base_azure_db_stage.yml [hunk 1]
```diff
@@ -11,42 +16,57 @@
...
-    - export PGUSER="${ADMIN_PG_USER}"
-    - export PGPASSWORD="${ADMIN_PG_PASSWORD}"
-    - create-db-user
+    GIT_STRATEGY: none
+    #
+    HELM_RENDER_ARGS: --set db.name=db_${CI_PIPELINE_ID}
+      --set db.password=pass_${CI_PIPELINE_ID}
+      --set db.user=user_${CI_PIPELINE_ID}
+      --set env[0].name=NEW_DB_EXTENSIONS
+      --set env[0].value=pgcrypto
```
Comment (revolunet) [inline base_azure_db_stage.yml:55]: there should be a better way

**Comment by revolunet** [comment]
the ephemeral DB are created using `CI_PIPELINE_ID` instead of CI_COMMIT_SHORT_SHA`

**Comment by revolunet** [comment]
cela fonctionne moyennant la recopie des secrets PG dans les nouvelles branches

**Comment by douglasduteil** [comment]
? la recopie des secrets PG dans les nouvelles branches ?
c'est dans ma meme pr ?

**Comment by revolunet** [comment]
non je ne sais pas où cela se fait

**Comment by revolunet** [comment]
will reopen later

**Pull request closed without merging**